initialization/dev.py

```diff
@@ -0,0 +1,39 @@
+import os
+import json
+from pathlib import Path
+
+
+def dump_project_metadata(app):
```

Comment:
This is just for my own development purposes, I will likely extract it into a general-purpose flask plugin.